packages/users/src/server/server.ts

```diff
@@ -57,32 +75,44 @@ export class UsersServer {
 
 	async deleteUser(user: User): Promise<void> {
 		try {
-			const where:string = "userId=\""+user.userId+"\"";
-			await UsersServer.dbserver.delete("users",where);
+			const queryOptions:QueryOptions = {
+				sql: "DELETE FROM users where userId=?"
+			}
+			const values:number[] = [user.userId];
+			await this.conn.query(queryOptions,values);
 		} catch (error) {
 			// TODO add user back if failed
-			// await UsersServer.dbserver.insert("users",[],[]);
+			await this.addUser(user);
 			console.error(error);
 		}
 	}
 
 	async createSession(username: string, password: string): Promise<Session> {
```

Comment:
E totusi o idee mai buna sa folosim null in loc de NO_SESSION pentru ca typescript va forta verificarea.

```suggestion
	async createSession(username: string, password: string): Promise<Session | null> {
```